docker-compose.yml

```diff
@@ -12,11 +12,12 @@ services:
     depends_on:
       - server-database
     environment:
-      - DATABASE_URL=server-database
+      - DATABASE_URL=mongodb://server-database:27017/ufood
       - PORT=8081
   server-database:
     image: mongo
     ports:
       - 27017:27017
+    # If running docker on windows, put in comments the volume because there is an incompatibility between mongo and windows volumes (you will always get connection refused)
```

Comment:
I would prefer to let the repo windows-agnostic :) Maybe we can add it in the documentation instead.

Reply:
Okay. I will remove the comment.